gmplot/gmplot.py

```diff
@@ -28,6 +28,8 @@ def safe_iter(var):
 
 class GoogleMapPlotter(object):
 
+    numMarkers = 0
```

Comment:
Can we define this closer to where it's used? If the same `GoogleMapPlotter()` object is used to draw twice, then `numMarkers` will be twice what it should be (this is just one example).

I'd recommend we add an `id` parameter to `write_point()`, then `write_points()` would define the count locally and pass the next marker ID to `write_point()`. If no ID is passed in (i.e. `id = None`), then `write_point()` would simply not give the rendered marker a name (as in `var marker_2 = ...`).